lib/api/dsl/storage/index.js

```diff
@@ -159,13 +180,13 @@ class Storage {
     addFiltersIndex(this.filtersIndex, index, collection, result.id);
 
     filters.forEach(sf => {
-      let
-        sfResult = addSubfilter(this.hashKey, this.filters[result.id], this.subfilters, sf),
-        addedConditions,
-        subfilter = this.subfilters[index][collection][sfResult.id];
+      const sfResult = addSubfilter(this.hashKey, this.filters[result.id], this.subfilters, sf);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 96 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)